spec/models/post.spec.js

```diff
@@ -73,5 +73,26 @@ describe("Post model", () => {
       })
     });
   })
+  it('can write a comment', function(done) {
+    const post = new Post({message: 'some message'});
 
+    post.save(function(err) {
+      expect(err).toBeNull();
+
+      Post.updateOne(
+          {'message': 'some message'},
+          {$set: {'comments': 'amazing'}},
+          function(err) {
+            expect(err).toBeNull();
+
+        Post.find(function(err, posts) {
+          expect(err).toBeNull();
+
+          expect(posts[0]).toMatchObject({
+            message: 'some message'});
```

Comment:
I suggest we need to add one more expect statement, something like:
expect(posts[0].comments[0]).toEqual('amazing');

Currently the unit test does not test for the comment itself

Reply:
sorted